app/org/sagebionetworks/bridge/models/accounts/SignUp.java

```diff
@@ -12,27 +12,20 @@
 
 public final class SignUp implements BridgeEntity {
 
-    private final String username;
     private final String email;
     private final String password;
     private final Set<Roles> roles;
     private final Set<String> dataGroups;
 
     @JsonCreator
-    public SignUp(@JsonProperty("username") String username, @JsonProperty("email") String email, 
-            @JsonProperty("password") String password, @JsonProperty("roles") Set<Roles> roles, 
-            @JsonProperty("dataGroups") Set<String> dataGroups) {
-        this.username = username;
+    public SignUp(@JsonProperty("email") String email, @JsonProperty("password") String password,
+            @JsonProperty("roles") Set<Roles> roles, @JsonProperty("dataGroups") Set<String> dataGroups) {
```

Comment:
Add a unit test where the JSON contains both username and email. This will ensure that when old clients pass in username, nothing breaks.